pkg/operator/ceph/object/rgw.go

```diff
@@ -245,6 +246,11 @@ func (c *clusterConfig) deleteStore() error {
 
 		err = deleteRealmAndPools(objContext, c.store.Spec)
```

Comment:
Let's go even further with changing this to a best-effort cleanup:
- `deleteStore()` should not even return an error. It should just log errors and continue. 
- The caller of `deleteStore()` in `object/controller.go` would not return an error to ever fail the reconcile
- No need to check for a specific failure like `InterruptSignal`. If there is any failure, just log the error and continue

Reply:
added these changes.